conventions.qmd

```diff
@@ -46,3 +46,31 @@ WILDS badges are maintained in a GitHub repository at <https://github.com/getwil
 
 An example badge: [![](https://getwilds.org/badges/badges/stable.svg)](https://getwilds.org/badges/#stable)
 
+### Contributing instructions
+
+All WILDS repos should have a `CONTRIBUTING.md` file. Although this file could live in many different places in a repository, in WILDS repositories it should live in the `.github/` directory.
+
+See also [GitHub's docs on CONTRIBUTING.md](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).
+
+### Pull request templates
+
+All WILDS repos should have one or more pull request templates. These help contributors include all the relavant information, and perform any required tasks, before submitting a pull request.
+
+See also [GitHub's docs on pull request templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)
+
+### Issue templates
+
+All WILDS repos should have one or more issue templates. These help contributors include all the relavant information, and perform any required tasks, before submitting an issue.
+
+See also [GitHub's docs on issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository)
+
+
+::: {.callout-warning}
+Question for the team: Should we have organization wide files (contributing, issues, PRs) for all WILDS repos? Or some other plan?
```

Comment:
Great question, I think it would be helpful to at least have a template available for people to use as a starting point, just so that they don't waste too much time thinking about it. Might just want to make it super clear that users should feel free to customize it as necessary since different repos might have different needs.

Reply:
Okay, yeah I think we should probably have org wide files so there's something there - probably make them pretty basic - and hopefully each maintainer will tailor it to their needs

Reply:
Full support

Reply:
opened a PR at https://github.com/getwilds/.github/pull/1